__tests__/app.test.js

```diff
@@ -0,0 +1,29 @@
+const request = require('supertest');
+const app = require('../app');
+const connection = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+
+
+
+beforeEach(() => seed(testData));
+afterAll(() => connection.end());
+
+describe("GET", () => {
+    describe("GET /api/topics", () => {
+        test('status code: 200 should return all topics with the appropriate properties', () => {
```

Comment:
Be careful with making claims in our test description for behaviours that aren't asserted for. This test is actually only checking that we get a 200, not checking against the appropriate properties as the description claims.